README.md

```diff
@@ -101,11 +98,10 @@ The context object also varies depending on the value being decorated. Breaking
   - `"field"`
   - `"accessor"`
 - `name`: The name of the value, or in the case of private elements the _description_ of it (e.g. the readable name).
-- `access`: An object containing methods to access the value on the instance.
+- `access`: An object containing methods to access the value. These methods also get the _final_ value of the private element on the instance, not the current value passed to the decorator. This is important for most use cases involving access, such as type validators or serializers. See the section on Access below for more details.
```

Comment:
Is this a normative change on the access object?

Reply:
No, this was how it always worked.